test/unit/test_dependencies.py

```diff
@@ -7,19 +7,20 @@
 def test_dbt_core_version() -> None:
     repo_root = pathlib.Path(__file__).parent.parent.parent
 
-    dbt_version_file = repo_root / "dbt" / "adapters" / "layer_bigquery" / "__version__.py"
-    with open(dbt_version_file) as f:
-        content = f.read()
-        match = re.search('version = "(.*)"', content)
-        if match:
-            dbt_version = match.group(1)
-        else:
-            raise Exception("Failed to get version")
+    for adapter in ["bigquery", "snowflake"]:
```

Comment:
Nit: parametrising the test is probably better than a for loop.

Reply:
Done!